rest/data/patch.go

```diff
@@ -79,6 +79,36 @@ func (pc *DBPatchConnector) FindPatchById(patchId string) (*restModel.APIPatch,
 	return &apiPatch, nil
 }
 
+func (pc *DBPatchConnector) FindPatchesByIds(patchIds []string) ([]restModel.APIPatch, error) {
+	mgobsonIds := []mgobson.ObjectId{}
+	for _, patchID := range patchIds {
+		if err := validatePatchID(patchID); err != nil {
+			return nil, errors.Wrap(err, "problem validating patchId")
+		}
+		mgobsonIds = append(mgobsonIds, mgobson.ObjectIdHex(patchID))
+	}
+
+	p, err := patch.Find(patch.ByIds(mgobsonIds))
+	if err != nil {
+		return nil, err
+	}
+	if p == nil {
+		return nil, errors.Wrap(err, "patches not found")
```

Comment:
If you get to this line, err will be nil. We should return our own error here with errors.New